Description: Remove rfc1320.txt from files to test
 rfc1320.txt is removed from the tarball so we need to ignore it in the test
 suite.
Author: Pierre Habouzit <madcoder@debian.org>
Forwarded: not-needed
Reviewed-By: Xavier Guimard <x.guimard@free.fr>
Last-Update: 2013-03-30

--- a/t/files.t
+++ b/t/files.t
@@ -5,7 +5,7 @@
         }
 }
 
-print "1..5\n";
+print "1..4\n";
 
 use strict;
 use Digest::MD4 qw(md4 md4_hex md4_base64);
@@ -26,7 +26,6 @@
 0dbd12438619d37abe39c41e4584ebe0  README
 ee852e2732fc4e158b1ca6b35e8f22da  MD4.pm
 3ae533df68e0be3ee2dac67329c8dad0  MD4.xs
-f178f04d0d8305c328a3de281313d642  rfc1320.txt
 EOT
 } elsif ("\n" eq "\015") { # MacOS
     # This is the output of: 'md4sum Changes README MD4.pm MD4.xs rfc1320.txt'
@@ -35,7 +34,6 @@
 a64a8e41ca2fe973ffbb46aa66d70bd2  README
 8f1d87025a08f7b1bdb968858d6c4044  MD4.pm
 6ef61bd88dfbb1f1401c2d8f3612d5fd  MD4.xs
-2089ab664427233cd7043d91f0021ff8  rfc1320.txt
 EOT
 } else {
     # This is the output of: 'md4sum Changes README MD4.pm MD4.xs rfc1320.txt'
@@ -44,7 +42,6 @@
 a64a8e41ca2fe973ffbb46aa66d70bd2  README
 8f1d87025a08f7b1bdb968858d6c4044  MD4.pm
 6ef61bd88dfbb1f1401c2d8f3612d5fd  MD4.xs
-2089ab664427233cd7043d91f0021ff8  rfc1320.txt
 EOT
 }
 
